cogs/channel_mods.py

```diff
@@ -875,116 +875,95 @@ async def modlog_delete(self, ctx, user, *, index):
-                    n_fnd_indx_str: Optional[List] = []  # define an empty list...
-                    n = 1
-                    for index in not_found_indexes:  # for every not found index...
-                        index = "#**" + index + "**,"  # format it (it's a string)...
-                        n_fnd_indx_str.append(index)  # append it to the empty list...
-                        n += 1
-                        if n > 10:  # up to 10 times...
-                            break
-                    n_fnd_indx_str = ' '.join(n_fnd_indx_str) # change the list to a string...
-                    summary_text = summary_text + f"\n**-** Invalid indexes: {n_fnd_indx_str[:-1]} and " \
-                                                  f"{len(not_found_indexes) - 10} more..."  # add it to the text.
-
-            if invalid_indexes:  # If invalid indexes were found (TypeError or <= 0):
-                invalid_indexes = sorted(invalid_indexes, key=str)  # sort them alphabetically (aesthetics)...
-                for index in invalid_indexes:  # Make it so that every str in the list has a single character:
-                    if len(index) > 1:
-                        x = invalid_indexes[invalid_indexes.index(index)].replace(index, index[0])
-                            # set a str equals to the first character of the str of a given index of the list
-                        invalid_indexes[invalid_indexes.index(index)] = x.lower()
-                            # make it lowercase (aesthetics) and set the element of that given index equals to it
-                if len(invalid_indexes) <= 10:  # same process as for not_found_indexes if they are less than ten.
-                    invalid_indexes = ["**" + i + "**," for i in invalid_indexes]
-                    inv_indx_str = ' '.join(invalid_indexes)
-                    summary_text = summary_text + f"\n**-** Invalid indexes: {inv_indx_str[:-1]}."
-                if len(invalid_indexes) > 10:  # And same process as for not_found_indexes if they are more than ten.
-                    inv_indx_str: Optional[list] = []
-                    n = 1
-                    for index in invalid_indexes:
-                        index = "**" + index + "**,"
-                        inv_indx_str.append(index)
-                        n += 1
-                        if n > 10:
-                            break
-                    inv_indx_str = ' '.join(inv_indx_str)
-                    summary_text = summary_text + f"\n**-** Invalid indexes: {inv_indx_str[:-1]} and " \
-                                                  f"{len(invalid_indexes) - 10} more..."
+            if removed_indices:  # If it removed logs in config:
+                removed_indices = ["#**" + i + "**" for i in removed_indices]  # format it...
+                rmv_indx_str = f"\n**-** Removed entries: {', '.join(removed_indices)}."  # change it to string...
```

Comment:
Note here I removed the comma from the end of the strings in the list and added it into the join command